CMakeLists.txt

```diff
@@ -112,7 +112,7 @@ endif()
 
 
 
-IF(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
+   IF(${CMAKE_SYSTEM_NAME} MATCHES "Linux" OR ${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
```

Comment:
Sounds like you like CMake PR but not the formatting one. I will separate them later.